internal/config/config.go

```diff
@@ -553,7 +553,7 @@ func (c *Config) RefreshOAuthToken(ctx context.Context, providerID string) error
 
 	switch providerID {
 	case string(catwalk.InferenceProviderAnthropic):
-		providerConfig.APIKey = fmt.Sprintf("Bearer %s", newToken.AccessToken)
```

Comment:
if we move this before the switch and remove it from line 559, we can close #1677